tests/test_biophysical/test_animal/test_data_types/test_beef_stocker_enums.py

```
@@ -0,0 +1,254 @@
+
+
+@pytest.mark.unit
+def test_stocker_diet_system_drylot_forage_value() -> None:
+    """StockerDietSystem.DRYLOT_FORAGE value must be 'drylot_forage'."""
+    assert StockerDietSystem.DRYLOT_FORAGE.value == "drylot_forage"
+
+
+@pytest.mark.unit
+def test_stocker_diet_system_invalid_raises_value_error() -> None:
+    """Constructing StockerDietSystem from an invalid string must raise ValueError."""
+    with pytest.raises(ValueError):
```

Ruff (0.15.21)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)